app/src/organisms/ODD/QuickTransferFlow/QuickTransferAdvancedSettings/DisposalVolume.tsx

```diff
@@ -65,7 +66,17 @@ export function DisposalVolume(props: DisposalVolumeProps): JSX.Element {
     if (typeof blowOut.location === 'string') {
       return blowOut.location
     }
-    return `trashBin:${blowOut.location.cutoutId}`
+    if (
+      'cutoutFixtureId' in blowOut.location &&
+      typeof blowOut.location.cutoutFixtureId === 'string' &&
+      WASTE_CHUTE_FIXTURES.includes(blowOut.location.cutoutFixtureId)
```

Comment:
do we not like this method? or you prefer checking the type before hand?

```suggestion
    if (
      'cutoutFixtureId' in blowOut.location &&
      WASTE_CHUTE_FIXTURES.includes(blowOut.location.cutoutFixtureId as string)
```

Reply:
not preference. for typescript, checking type is better than using assertion.